### ISSUE TITLE
cloud: caching does not work if there are fragments

### ISSUE THREAD

**Issue by shahidhk** [hasura/graphql-engine#7031]
version: `v2.0.0-cloud.11`

Using `@cached` directive does not cache the request if the query has fragments. 

Eg:

This query gets cached:

```gql
query getTest @cached {
  test {
    name
  }
}
```

But this one doesn't:
```gql
query getTest @cached {
  test {
    ...gist
  }
}
fragment gist on test {
  name
}
```

**Comment by ssbothwell** [comment]
This is resolved by PR #1523 in the monorepo.

**Comment by ssbothwell** [comment]
hasura/graphql-engine-mono#1523 has been merged so I am closing this issue.
